### ISSUE TITLE
Not pulling the full URL?

### ISSUE THREAD

**Issue by MikeTannenbaum** [Blogger-Peer-Review/quotebacks#21]
Hey hey, I'm surprised by this one since it's worked on 99% of the 10+ snippets I've saved, so I'm guessing it's something to do with the original source's website. BUT, a snippet I clip truncates the URL of the sources to the TLD, not the actual slug. Can't figure out why so thought I'd share with ya.

https://www.tolerance.org/magazine/ally-or-accomplice-the-language-of-activism

clip something on that page and, for me, when I click "go to text" it'll take me to the main .org site instead of the actual article.

Thanks for making this! Loving the tiny uses I've been experimenting with thus far.

**Comment by tomcritchlow** [comment]
This is a hard edge case. We respect the canonical link declared on the page - to try and remove tracking IDs and "URL cruft". Unfortunately - this URL has an incorrect canonical tag declared that points to the homepage.

Not sure what is best for us here:

a) ignore the canonical tag everywhere and end up with more "ugly" URLs with tracking IDs etc.

b) respect the canonical tag but end up with edge cases like this where they have broken the links themselves....

**Comment by MikeTannenbaum** [comment]
based on how you laid that out I'd leave this as an edge case.

that said, the ability to edit the URL once it's been snipped would be useful. I tried to go in and manually change the source URL but couldn't find it. 

Food for thought!

**Comment by tomcritchlow** [comment]
Yeah - we don't have that ability in the extension... You can change it after you copy the embed but that's not exactly what you're after....
